### PR TITLE
Small fixes and moved reset_equalizer to disconnect

### PR DESCRIPTION
**Pull request by Zenrac**
### Checks and guidelines:

* [x] Have you checked that there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Have you checked that only **single quotes** are used in the code, apart from the doc-strings?
* [x] Have you run a lint program on your code prior to submission?
* [x] Have you checked if `python run_tests.py` returns no errors?

Tests output: 
image

### Type of change

* [ ] Bug fix
* [x] Lint fix
* [ ] New feature
* [x] Improvement
* [ ] Breaking change
* [ ] This change is a documentation update

### Describe the changes:

- Small fixes from last commit (do you even lint bro ?!) 
- Moving `player.reset_equalizer()` to `player.disconnect()` is this a good idea ?

### CODE REVIEW

File: lavalink/Client.py [hunk 1]
```diff
@@ -17,8 +17,8 @@ def set_log_level(log_level):
 
 
 class Client:
-    def __init__(self, bot, log_level = logging.INFO, loop = asyncio.get_event_loop(), default_node: int = 0,
-                 player = DefaultPlayer, rest_round_robin: bool = False):
+    def __init__(self, bot, log_level=logging.INFO, loop=asyncio.get_event_loop(), default_node: int = 0,
```
Comment (Devoxin) [inline lavalink/Client.py]: Why was this changed? It's no longer consistent with the rest :(

File: lavalink/Client.py [hunk 2]
```diff
@@ -17,8 +17,8 @@ def set_log_level(log_level):
 
 
 class Client:
-    def __init__(self, bot, log_level = logging.INFO, loop = asyncio.get_event_loop(), default_node: int = 0,
-                 player = DefaultPlayer, rest_round_robin: bool = False):
+    def __init__(self, bot, log_level=logging.INFO, loop=asyncio.get_event_loop(), default_node: int = 0,
+                 player=DefaultPlayer, rest_round_robin: bool = False):
```
Comment (Devoxin) [inline lavalink/Client.py]: and this

File: lavalink/Client.py (hunk 1 above)
Comment (Zenrac) [reply lavalink/Client.py]: I assume that PEP8 want spaces when you specify a type and no spaces when you don't

Comment (Devoxin) [reply lavalink/Client.py]: Ah, would it be worth adding types?

Comment (Zenrac) [reply lavalink/Client.py]: I don't think so. 

Otherwise we could add an exception to flake8 to avoid error with travis/run_tests.py

Comment (Devoxin) [reply lavalink/Client.py]: eh, nevermind it's fine as-is
